### ISSUE TITLE
Is this still maintained, or show we use another fork?

### ISSUE THREAD

**Issue by phil-hudson** [dwarvesf/hidden#284]
title - looks like no releases for a few years?

**Comment by edofe99** [comment]
It has several bug unresolved since may years. It's not maintained anymore.

**Comment by scalzadonna** [comment]
any alternative to recommend?
I used to have Dozer, but it's also discontinued

**Comment by edofe99** [comment]
> any alternative to recommend? I used to have Dozer, but it's also discontinued

Bartender 5 is the best, it's paid unfortunately. It's the one that has less bugs.

**Comment by leoravagnani** [comment]
@phil-hudson @scalzadonna @ironhak I asked the same thing a while ago at #251 and a good soul chimed in with a fork!! take a look at https://github.com/UeharaYou/HiddenBar

**Comment by amarendra** [comment]
> take a look at https://github.com/UeharaYou/HiddenBar

I don't think that even has an installer. So that leaves out most users.

**Comment by leoravagnani** [comment]
> > take a look at https://github.com/UeharaYou/HiddenBar
> 
> I don't think that even has an installer. So that leaves out most users.

Well, I just stumbled upon this: https://github.com/jordanbaird/Ice
It has some bugs still, but looks promising with more features and is actively maintained.

**Comment by UeharaYou** [comment]
> > > take a look at https://github.com/UeharaYou/HiddenBar
> > 
> > 
> > I don't think that even has an installer. So that leaves out most users.
> 
> Well, I just stumbled upon this: https://github.com/jordanbaird/Ice It has some bugs still, but looks promising with more features and is actively maintained.

Yes that's way better than my fork, the API it uses is way more powerful than HiddenBar.

**Comment by amarendra** [comment]
> Well, I just stumbled upon this: https://github.com/jordanbaird/Ice

The screen recording part makes me uncomfortable. I know it has a banner saying it doesn't record anything and I guess it doesn't. But I don't know much about it. Battery implications etc (which I am not sure of). But I am just uncomfortable. I wish it worked without that permission. But I can't even see the settings screen unless both the permissions are given. So I had to uninstall it. I will just use Dozer (though its shortcut doesn't work) or stop using any Menu hider. Thanks for the suggestion though.

**Comment by pokono** [comment]
Are you guys looking for maintainers? Would be great to maintain the official version with the official philosophy, more than fork away!

**Comment by ElhemEnohpi** [comment]
> > Well, I just stumbled upon this: https://github.com/jordanbaird/Ice
> 
> The screen recording part makes me uncomfortable. I know it has a banner saying it doesn't record anything and I guess it doesn't. But I don't know much about it. Battery implications etc (which I am not sure of). But I am just uncomfortable. I wish it worked without that permission. But I can't even see the settings screen unless both the permissions are given. So I had to uninstall it. I will just use Dozer (though its shortcut doesn't work) or stop using any Menu hider. Thanks for the suggestion though.

Ice does work without screen recording permission, for anyone reading this... It can do a couple of more things if you give it the permission, but it's not required for basic operation. For example, you can't rearrange the menu items from within Ice's settings screen without it, but you can just do that directly in the menu bar by command-drag. See:

https://github.com/jordanbaird/Ice/issues/598

**Comment by MrSaneApps** [comment]
Late to this thread, but for anyone still looking: I built [SaneBar](https://sanebar.com) as a menu bar manager that doesn't require screen recording permission. Works on macOS 15 and Tahoe, open source, actively maintained.

**Comment by tieubao** [comment]
Yes, Hidden Bar is actively maintained again as of 2026. Recent fixes are on the `develop` branch and a v1.11 release is being prepared. No need for a fork. Closing, thanks for checking in.
